notebooks/ConsPrefShockModel.py

```diff
@@ -0,0 +1,267 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.2'
+#       jupytext_version: 1.2.4
+#   kernelspec:
+#     display_name: Python 2
+#     language: python
+#     name: python2
+# ---
+
+# %%
+# Initial imports and notebook setup, click arrow to show
+import sys 
+import os
+import numpy as np
+
+from HARK.ConsumptionSaving.ConsPrefShockModel import *
```

Comment:
`*` imports aren't encouraged as they pollute the namespace, can you change this to something like
`from HARK.ConsumptionSaving.ConsPrefShockModel import PrefShockConsumerType, KinkyPrefConsumerType`